data/xapleaux/xapleaux outfits.txt

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2021 by MasterOfGrey
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+outfit "String Drive"
+	category Systems
+	cost 3500000
+	mass 15
+	thumbnail "outfit/string drive"
+	"energy consumption" 2.5
+	hyperdrive 1
+	"jump fuel" 50
+	"outfit space" -15
+	"scram drive" 1.5
+	description "A string drive constantly hums with the frequency of space-time in its vicinity. By constantly remaining attuned to the threads that link one system to the next it has no need to slow down enough to 'search' for a path to follow. Simply by aligning in the correct direction it can claw its way along any hyperthread in its vicinity even if your ship is traveling at a very significant speed. Consequently it needs very little fuel to initiate the process and is smaller as a result. However, this comes at a cost of continuously draining quite a bit of energy to power the attunement apparatus."
```

Comment:
I would like to suggest that a hyperlane, as a gravitational slipstream, essentially "frays" at the ends. That being part of why you have to stabilise enough to lock onto it with a hyperdrive instead of simply "falling into" a gravity slipstream.

In this case, the implication is that it is constantly attuned to the state of those frayed ends, so it doesn't have to pull them together or hunt for them, it can just grab the nearest thread it's aware of and pull itself into the hyperlane from there.

It's getting a bit esoteric about the physics.

Reply:
Will the player find out about hyperthreads at some point when in Xapelaux space? This seems like important information about a central concept in Endless Sky that should probably be elaborated on at some point.

Reply:
If I get my way they will, and if not, I can always switch the description back - at present there's not going to be any way for the player to see or acquire the drive until after I progress the Xapleaux plot far enough that I'll know whether hyperthreads are going to be a viable lore-element or not, so having it in serves as a harmless and useful reminder given the timeline of development.

Reply:
```suggestion
	description "A string drive continuously hums with the frequency of space-time in its vicinity. By constantly remaining attuned to the threads that link one system to the next it has no need to slow down enough to 'search' for a path to follow. Simply by aligning in the correct direction it can claw its way along any hyperthread in its vicinity even if your ship is traveling at a very significant speed. Consequently it needs very little fuel to initiate the process and is smaller as a result. However, this comes at a cost of continuously draining quite a bit of energy to power the attunement apparatus."
```